### PR TITLE
Extract react scripts

### LINKED ISSUE #7: Make the helper scripts their own package
This will clean up a bit and make them easier to reason about.

They can then also be used in other projects.

### PR DESCRIPTION
**Pull request by asbjornh**
Fixes #7

### CODE REVIEW

**Comment by asbjornh** [comment]
Remove `creuna-new` cli command and remove `@creuna/react-scripts` as a local dependency

**Comment by pellebjerkestrand** [comment]
Is this good to go now or do you want/need to do more work on it?

File: bin/index.js [hunk 1]
```diff
@@ -1,4 +1,19 @@
...
 
-newProject(process.argv[2]);
+const blue = chalk.blueBright;
+const cyan = chalk.cyan;
+
+console.log(
+  `🤗  Hey! ${blue('creuna-new')} has been deprecated. Use ${blue(
+    '@creuna/cli'
+  )} instead! It's much nicer.
+  • ${blue('yarn remove -g @creuna/create-react-app')} or ${cyan(
```
Comment (pellebjerkestrand) [inline bin/index.js]: Global in Yarn is done using `yarn global <command>` not with a `-g` flag.

Comment (asbjornh) [reply bin/index.js]: Fixed

**Pull request merged**